src/Xlsx.php

```diff
@@ -524,4 +524,14 @@ private function replaceValue($value)
             return false;
         }
     }
+
+    /**
+    *duplicateStyle
+    */
+    public function duplicateStyle($cellStyle = null, $range = '')
```

Comment:
## Point 2. I want the same as other API

For example: How to select a cell ( see https://github.com/k1LoW/xlsx/pull/5/files#diff-8fbaad7573693e481094e6665e64d975L216 )